src/adapters/demo-result.mapper.ts

```diff
@@ -0,0 +1,45 @@
+import { EmpathyResult } from '@empathy/search-adapter';
+import { Result } from '@empathy/search-types';
+
+type CustomResult = {
+  gender: string;
+  color: string;
+  year: string;
+  categoryPaths: string[];
+  usage: string;
+  groupId: string;
+  type: string;
+  score: number;
+  categoryIds: string[];
+  size: string;
+  season: string;
+  categories: string[];
+};
+
+/**
+ * This mapper adapts the response from the api to X Components.
+ *
+ * @param rawResult - This is the raw response form the API.
+ * @param result - This is the Empathy result handle by the app.
+ *
+ * @returns A new object which can be handled by XComponents.
+ */
+export function resultMapper(
```

Comment:
Why is this mapper necessary? We are not using any of these values. Do we really need it?

I'd remove it.

Reply:
I think in a near future we're going to style the results and shows more information, this properties are just the ones available.

We can remove them but later we're going to need them again I believe.